platform/javascript/js/libs/library_godot_javascript_singleton.js

```diff
@@ -0,0 +1,333 @@
+		if (obj === undefined) {
+			return;
+		}
+		const name = GodotRuntime.parseString(p_name);
+		try {
+			obj[name] = GodotJSWrapper.variant2js(p_type, p_exchange);
+		} catch (e) {
+			GodotRuntime.error(`Error setting variable ${name} on object`, obj);
+		}
+	},
+
+	godot_js_wrapper_object_call__sig: 'iiiiiiiii',
+	godot_js_wrapper_object_call: function (p_id, p_method, p_args, p_argc, p_convert_callback, p_exchange, p_lock, p_free_lock_callback) {
+		const obj = GodotJSWrapper.get_proxied_value(p_id);
+		if (obj === undefined) {
+			return -1;
+		}
+		const method = GodotRuntime.parseString(p_method);
+		const convert = GodotRuntime.get_func(p_convert_callback);
+		const freeLock = GodotRuntime.get_func(p_free_lock_callback);
+		const args = new Array(p_argc);
+		for (let i = 0; i < p_argc; i++) {
+			const type = convert(p_args, i, p_exchange, p_lock);
+			const lock = GodotRuntime.getHeapValue(p_lock, '*');
+			args[i] = GodotJSWrapper.variant2js(type, p_exchange);
+			if (lock) {
+				freeLock(p_lock, type);
+			}
+		}
+		try {
+			const res = obj[method](...args);
+			return GodotJSWrapper.js2variant(res, p_exchange);
+		} catch (e) {
+			GodotRuntime.error(`Error calling method ${method} on:`, obj, 'error:', e);
+			return -1;
+		}
+	},
+
+	godot_js_wrapper_object_unref__sig: 'vi',
+	godot_js_wrapper_object_unref: function (p_id) {
+		const proxy = IDHandler.get(p_id);
+		if (proxy !== undefined) {
+			proxy.unref();
+		}
+	},
+
+	godot_js_wrapper_create_cb__sig: 'vii',
+	godot_js_wrapper_create_cb: function (p_ref, p_func) {
+		const func = GodotRuntime.get_func(p_func);
+		let id = 0;
+		const cb = function () {
+			if (!GodotJSWrapper.get_proxied_value(id)) {
+				return;
+			}
+			const args = Array.from(arguments);
+			func(p_ref, GodotJSWrapper.get_proxied(args), args.length);
+		};
+		id = GodotJSWrapper.get_proxied(cb);
+		return id;
+	},
+
+	godot_js_wrapper_object_getvar__sig: 'iiii',
+	godot_js_wrapper_object_getvar: function (p_id, p_type, p_exchange) {
+		const obj = GodotJSWrapper.get_proxied_value(p_id);
+		if (obj === undefined) {
+			return -1;
+		}
+		const prop = GodotJSWrapper.variant2js(p_type, p_exchange);
+		if (prop === undefined || prop === null) {
+			return -1;
+		}
+		try {
+			return GodotJSWrapper.js2variant(obj[prop], p_exchange);
+		} catch (e) {
+			GodotRuntime.error(`Error getting variable ${prop} on object`, obj, e);
+			return -1;
+		}
+	},
+
+	godot_js_wrapper_object_setvar__sig: 'iiiiii',
+	godot_js_wrapper_object_setvar: function (p_id, p_key_type, p_key_ex, p_val_type, p_val_ex) {
+		const obj = GodotJSWrapper.get_proxied_value(p_id);
+		if (obj === undefined) {
+			return -1;
+		}
+		const key = GodotJSWrapper.variant2js(p_key_type, p_key_ex);
+		try {
+			obj[key] = GodotJSWrapper.variant2js(p_val_type, p_val_ex);
+			return 0;
+		} catch (e) {
+			GodotRuntime.error(`Error setting variable ${key} on object`, obj);
+			return -1;
+		}
+	},
+
+	godot_js_wrapper_create_object__sig: 'iiiiiiii',
```

Comment:
That would be more readable as `viii` ;) (ok I'm pretty sure I already made that joke last year :P)